docs/install/upgrade-installation.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Upgrading your installation"
+weight: 21
+type: "docs"
+---
+
+If you already have Knative installed, upgrade your installation by running the
+install command for the new version. We recommend upgrading by a single version
+number. For example, if you have v0.6.0 installed, you should upgrade to v0.7.0
```

Comment:
Thinking about this more, I think we also need to clarify that the "upgrade" install command should install the next release of both the component and their related plugins (ie. make sure that the upgraded Serving gets the same supported version of "monitoring" for example, etc.).

Reply:
done.